src/content/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration.mdx

```diff
@@ -239,6 +239,243 @@ Environmental variables allow you to fine-tune the synthetics job manager config
     </table>
   </Collapser>
 
+  <Collapser
+    id="podman-env-config"
+    title="Podman environment configuration"
+  >
+    The variables are provided at startup using the `-e, --env` argument.
+
+    The following table shows all the environment variables that synthetics job manager supports. `PRIVATE_LOCATION_KEY` is required, and all other variables are optional. To have the synthetics job manager run in a Podman environment, the minimum version should be release-418  or higher.
+
+    <table>
+      <thead>
+      <tr>
+        <th>
+          Name
+        </th>
+
+        <th>
+          Description
+        </th>
+      </tr>
+      </thead>
+
+      <tbody>
+      <tr>
+        <td>
+          `PRIVATE_LOCATION_KEY`
+        </td>
+
+        <td>
+          <DNT>**REQUIRED.**</DNT> Private location key, as found on the Private Location entity list.
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          `HORDE_API_ENDPOINT`
+        </td>
+
+        <td>
+          For US-based accounts, the endpoint is: `https://synthetics-horde.nr-data.net.`
+
+          For [EU-based](/docs/using-new-relic/welcome-new-relic/get-started/introduction-eu-region-data-center#partner-hierarchy) accounts, the endpoint is: `https://synthetics-horde.eu01.nr-data.net/`
+
+          Ensure your synthetics job manager can connect to the appropriate endpoint in order to serve your monitor.
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          `PODMAN_API_SERVICE_HOST`
+        </td>
+
+        <td>
+          The host entry added to the Pod created where the SJM is going to run. Use this to override `podman.service` as the default.
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          `PODMAN_API_SERVICE_PORT`
+        </td>
+
+        <td>
+          The port at which the Podman LibPod RESTful API service is running in the instance. Use this to override `8000` as the default.
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          `PODMAN_API_VERSION`
+        </td>
+
+        <td>
+          The specific version of the Podman LibPod RESTful API being used. Use this to override `v5.0.0` as the default.
```

Comment:
what happens when this version is overridden?

Reply:
that version what ever is mentioned will be used in the URL we build. The minimum podman version specified uses the v5.0.0 which has all the endpoints support required by our SJM.